arch/arm/boot/dts/overlays/Makefile

```diff
@@ -169,6 +170,7 @@ dtbo-$(CONFIG_ARCH_BCM2835) += \
 	spi6-2cs.dtbo \
 	ssd1306.dtbo \
 	ssd1306-spi.dtbo \
+	ssh1351-spi.dtbo \
```

Comment:
This should be ssd1306-spi.dtbo

Reply:
Thanks, but in fact it shall be ssd1351-spi.dtbo ;)